code/modules/mob/living/carbon/xenomorph/Embryo.dm

```diff
@@ -111,15 +114,19 @@
 		if(4)
 			if(prob(1))
 				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_BONE_BREAK)
```

Comment:
i'm fairly sure this is a misuse of that define, you should probably add a PAIN_CHESTBURSTING define